### ISSUE TITLE
Please add OS X support

### ISSUE THREAD

**Issue by ahrenstein** [neillturner/kitchen-ansible#98]
Using the vmware_fusion provider and OS X vagrant boxes, Ansible does not work.

```
Transferring files to <default-mavericks>
       Going to invoke ansible-playbook with: sudo -E PATH=/tmp/kitchen/ansible/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games PYTHONPATH=/tmp/kitchen/ansible/lib MANPATH=/tmp/kitchen/ansible/docs/man /tmp/kitchen/ansible/bin/ansible-playbook --inventory-file=/tmp/kitchen/inventory -c local -M /tmp/kitchen/modules -vvv       /tmp/kitchen/site.yml
       sudo: /tmp/kitchen/ansible/bin/ansible-playbook: command not found
>>>>>> Converge failed on instance <default-mavericks>.
>>>>>> Please see .kitchen/logs/default-mavericks.log for more details
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: SSH exited (1) for command: [sudo -E PATH=/tmp/kitchen/ansible/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games PYTHONPATH=/tmp/kitchen/ansible/lib MANPATH=/tmp/kitchen/ansible/docs/man /tmp/kitchen/ansible/bin/ansible-playbook --inventory-file=/tmp/kitchen/inventory -c local -M /tmp/kitchen/modules -vvv       /tmp/kitchen/site.yml]
```

It seems ansible is not copied over to the system

**Comment by neillturner** [comment]
it doesn't support OSX so this would be an enhancement.

**Comment by trinitronx** [comment]
@ahrenstein: This is more of a generic issue with the high barrier to getting a working OS X vagrant box, since there is no official public one (probably because the OS X licence agreement only allows running OS X VMs on already purchased Mac hardware, and everyone's scared of posting one publicly and trusting people to only run it there).  I have tried in the past to get this working for various reasons, and was only able to with `vmware_fusion`, and on an older Macbook which I no longer possess.  Anyway, this is a long-winded way of saying that I haven't had the time to create a new working OS X vagrant box & get something working and tested, although for another home project I do need such a box.

Practically speaking, there is a way to solve the problem.

Ansible is on Homebrew, so it's just a matter of detecting OS X as a platform, getting homebrew installed & running `brew install ansible`.

**Comment by trinitronx** [comment]
Apparently since [I tried last on VirtualBox][1], there has been some development on other fronts:  

 - [HowTo: Build an OSX 10.10 base-box for test-kitchen and VirtualBox][2]
 - chef/bento#353

[1]: https://github.com/jedi4ever/veewee/issues?utf8=%E2%9C%93&q=+author%3Atrinitronx+
[2]: https://gist.github.com/rmoriz/37b671afe53c984b2f85

**Comment by ahrenstein** [comment]
Yeah I had to create my own boxes for this. I think we can leave this as resolved since pull request #99 allowed me to get Ansible working
